binder/environment.yml

```
@@ -16,3 +16,4 @@ dependencies:
   - autopep8
   - conda-forge::jupyter_contrib_nbextensions
   - pymc3
+  - sklearn
```

```suggestion
  - scikit-learn
```
Binder build was failing with "nothing provides requested sklearn". `pip install sklearn` does seem to work, but docs suggest `pip install scikit-learn` so also suggested changing that in requirements.txt